Postman/dex.postman_collection.json

```diff
@@ -2960,6 +3087,122 @@
 							],
 							"protocolProfileBehavior": {},
 							"_postman_isSubFolder": true
+						},
+						{
+							"name": "File",
+							"item": [
+								{
+									"name": "Post-File-Administrator",
+									"request": {
+										"method": "POST",
+										"header": [
+											{
+												"key": "IdentityId",
+												"value": "{{administratorUserIdentityId}}",
+												"type": "text"
+											}
+										],
+										"body": {
+											"mode": "formdata",
+											"formdata": [
+												{
+													"key": "File",
+													"type": "file",
+													"src": "/C:/Users/Niray/Desktop/angular-226066.png"
```

Comment:
My postman tests don't pass: https://gyazo.com/8bec3b63972699b60b544264b2ad96a2 
I reckon this is because of above mentioned hardcoded file.

Reply:
Should pass now.